specification/glossary.md

```diff
@@ -104,6 +104,10 @@ A company or organization that provides outsourced management and support of a r
 
 A term that describes a service that is available and provided immediately or as needed, without requiring a pre-scheduled appointment or prior arrangement. In cloud computing, virtual machines can be created and terminated as needed, i.e. on demand.
 
+<a name="glossary:negotiated-discount"><b>Negotiated Discount</b></a>
+
+A contractual agreement where a customer commits to specific spend or usage goals over a [*term*](#glossary:term) in exchange for discounted rates across varying SKUs.  Unlike [*commitment discounts*](#glossary:commitment-discount), negotiated discounts are typically more customized to customer's accounts, can be burned down at varying frequencies, and may overlap with *commitment discounts*.
```

Comment:
```suggestion
A contractual agreement where a customer commits to specific spend or usage goals over a [*term*](#glossary:term) in exchange for discounted rates across varying SKUs.  Unlike [*commitment discounts*](#glossary:commitment-discount), negotiated discounts are typically more customized to customer's accounts, can be utilized at varying frequencies, and may overlap with *commitment discounts*.
```

**utilized** is used in [FinOps terminology](https://www.finops.org/assets/terminology/) definition and covers both build-up and burn-down use cases, whereas **burned down** excludes build-up negotiated discounts - EDP, MACC, etc.